opta/core/kubernetes.py

```diff
@@ -343,13 +343,18 @@ def tail_module_log(
     v1 = CoreV1Api()
     watch = Watch()
     count = 0
+    """Using the UTC Time stamp as the Kubernetes uses the UTC Timestamps."""
+    start_time = datetime.datetime.utcnow()
     for event in watch.stream(
         v1.list_namespaced_pod,
         namespace=layer.name,
         label_selector=f"app.kubernetes.io/instance={layer.name}-{module_name}",
     ):
         pod: V1Pod = event["object"]
         color_idx = count % (256 - start_color_idx) + start_color_idx
+        if compare_datetime(start_time, pod.metadata.creation_timestamp) == -1:
```

Comment:
are you sure pod.metadata.creation_timestamp has timezone?

Reply:
They have a default Timezone of UTC0. But it can be changed. Let me check on how I can get the timezone of the Pod.